code/modules/antagonists/clockcult/clock_scriptures/scripture_drivers.dm

```diff
@@ -216,6 +216,9 @@
 	if(is_reebe(invoker.z))
 		to_chat(invoker, "<span class='danger'>You're already at Reebe.</span>")
 		return
+	if(invoker.z == 0)
```

Comment:
```suggestion
	if(!isturf(invoker.loc))
```
I never recalled z, x and y coordinates being affected by containers in such way.
Regardless, this should be in line with the other hundreds of "must be on a turf" sort of checks.